src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -20560,4 +20560,261 @@ def test_aks_jwtauthenticator_cmds(self, resource_group, resource_group_location
             "aks jwtauthenticator list --resource-group={resource_group} --cluster-name={name} "
         )
         jwt_list_after_delete = self.cmd(list_after_delete_cmd).get_output_in_json()
-        assert len(jwt_list_after_delete) == 0
\ No newline at end of file
+        assert len(jwt_list_after_delete) == 0
+
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(
+        random_name_length=17, name_prefix="clitest", location="eastus"
+    )
+    def test_aks_nodepool_get_rollback_versions(self, resource_group, resource_group_location):
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=140860864&view=results) to validate the change.

Reply:
Latest live test passed https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=141396983&view=logs&j=b162b355-d59d-5864-ce0f-0a70f12dd28b&t=b162b355-d59d-5864-ce0f-0a70f12dd28b